src/makeInfinityAdder.js

```diff
@@ -3,8 +3,26 @@
 /***
  * @return {function}
  */
-function makeInfinityAdder() {
-  // write code here
+function makeInfinityAdder(num) {
+  let sum = num || 0;
```

Comment:
The `sum` should be initialized to 0 within the `result` function, not in the `makeInfinityAdder` function. This is because each call to `makeInfinityAdder` should create a new adder function with its own closure over `sum`.